ictv/renderer/renderer.py

```diff
@@ -183,19 +190,9 @@ def __init__(self, *args, **kwargs):
                 templates[os.path.splitext(template)[0]] = {}
 
         for template in templates:
-            def f(type):
-                def g(*args, **kwargs):
-                    id = type + '-' + str(kwargs['number'])
-                    templates[template][id] = {'max_chars': kwargs['max_chars']} if 'max_chars' in kwargs else {}
-                return g
-
-            dummy_renderer = SlideRenderer({'title': f('title'), 'subtitle': f('subtitle'),
```

Comment:
Was this unused ?

Reply:
Yep, it was needed before because with web.py we needed to render the template in order to retrieve variables from it. But it is enough to parse the file with jinja so now this part is useless